src/components/TextLink/TextLink.module.scss

```diff
@@ -1,6 +1,4 @@
 .text-link {
-  font-size: inherit;
```

Comment:
Inheriting font size is already the default behavior, so the style is redundant, but also prevents usage of css utility classes